pkg/runner/cluster_k8s.go

```diff
@@ -556,6 +556,7 @@ func (c *ClusterK8sRunner) createPod(ctx context.Context, podName string, input
 				"testground.testcase": runenv.TestCase,
 				"testground.run_id":   input.RunID,
 				"testground.groupid":  g.ID,
+				"testground.purpose":  "plan",
```

Comment:
I used the same label here as I did in https://github.com/ipfs/testground/pull/581

but I am open to doing this another way, if we don't like this.

Reply:
I think this is fine. We might want to introduce a filter for all things Testground related, such as `app=testground`, but we definitely need to differentiate between testground-related services (redis, sidecar, etc.) and testplan instances, so this filter could be used for that.